Model/Data/AmwalOrderDetails.php

```diff
@@ -57,33 +67,10 @@ public function updateOrderStatus()
         }
         $status = $amwalOrderData['status'];
```

Comment:
```suggestion
```

Reply:
Here we use the $status in $historyComment